makefile

```diff
@@ -23,23 +23,27 @@ filenamesc := ft_strlen.c \
 			ft_strncmp.c \
 			ft_atoi.c \
```

Comment:
Is placing this on multiple lines mostly an aesthetic choice? Either way I can dig it.